### PR TITLE
print warnings if head position contains not-a-number

### PR DESCRIPTION
**Pull request by sethalves**
- print warnings if head position contains not-a-number

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6616/app_12264_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6616/HighFidelity-Beta-PR12264-b6f7d2eb892aa0c669a5d90a3f4822a1ff9d6476.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6616/HighFidelity-Beta-PR12264-b6f7d2eb892aa0c669a5d90a3f4822a1ff9d6476.exe

**Comment by sethalves** [comment]
to test:
- should be no functional change.
- pay attention to the avatar and other avatars, watch for anomalies
- try 3rd person camera and mirror camera

File: interface/src/avatar/MyAvatar.cpp [hunk 1]
```diff
@@ -2700,27 +2706,42 @@ void MyAvatar::setWalkSpeed(float value) {
 }
 
 glm::vec3 MyAvatar::getPositionForAudio() {
+    glm::vec3 result;
     switch (_audioListenerMode) {
         case AudioListenerMode::FROM_HEAD:
-            return getHead()->getPosition();
+            result = getHead()->getPosition();
```
Comment (hyperlogic) [inline interface/src/avatar/MyAvatar.cpp]: you need to add break;

File: interface/src/avatar/MyAvatar.cpp [hunk 2]
```diff
@@ -2700,27 +2706,42 @@ void MyAvatar::setWalkSpeed(float value) {
...
+    return result;
 }
 
 glm::quat MyAvatar::getOrientationForAudio() {
+    glm::quat result;
+
     switch (_audioListenerMode) {
         case AudioListenerMode::FROM_HEAD:
-            return getHead()->getFinalOrientationInWorldFrame();
+            result = getHead()->getFinalOrientationInWorldFrame();
```
Comment (hyperlogic) [inline interface/src/avatar/MyAvatar.cpp]: you need to add break;

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6635/app_12264_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6635/HighFidelity-Beta-PR12264-3be42fe79aae5840e08e2166f88fbe9147f2def5.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6635/HighFidelity-Beta-PR12264-3be42fe79aae5840e08e2166f88fbe9147f2def5.dmg

**Comment by sbennett77** [comment]
Currently Testing.

**Comment by sbennett77** [comment]
Passed.

@conklin94122

**Pull request merged**